src/au/aulayer.cpp

```diff
@@ -1,4 +1,7 @@
 
+#include "SurgeLogger.h"
+
+
```

Comment:
Why two extra newlines here?

Reply:
Do we have any convention how to order includes?

Reply:
The newlines are sloppy and I will remove them but the whole allayed.cpp code is a mix of carbon and cocoa and ancient and new and this tab and that tab, so I will do a separate PR to generally clean it up.